pydantic_ai_slim/pydantic_ai/low_level.py

```diff
@@ -0,0 +1,234 @@
+"""Low-level methods to make requests directly to models with minimal abstraction.
+
+These methods allow you to make requests to LLMs where the only abstraction is input and output schema
+translation so you can request all models with the same API.
+
+These methods are thin wrappers around [`Model`][pydantic_ai.models.Model] implementations.
+"""
+
+from __future__ import annotations as _annotations
+
+import dataclasses
+from collections.abc import AsyncIterator
+from contextlib import asynccontextmanager
+
+from pydantic_graph._utils import get_event_loop as _get_event_loop
+
+from . import agent, messages, models, settings, usage
+from .models import instrumented as instrumented_models
+
+__all__ = 'LowLevelModelResponse', 'model_request', 'model_request_sync', 'model_request_stream'
+
+
+@dataclasses.dataclass
+class LowLevelModelResponse(messages.ModelResponse):
+    """Subclass of [`ModelResponse`][pydantic_ai.messages.ModelResponse] that includes usage information."""
```

Comment:
I'm still not sure why the usage isn't included in ModelResponse. Is it because people might misinterpret it to mean the usage of the whole agent run? What if it was called `request_usage`? People might actually want to access that info via `agent_run.all_messages()`

Reply:
Otherwise does this name need to be tied to the module? What about `ModelResponseWithUsage`?

Reply:
ye, we can probably change it everywhere. Now I see it's the usage for just that request, I think it makes more sense to just add it to `ModelResponse`